File: scrape_ainex.js
const { chromium } = require('playwright');
const fs = require('fs');

(async () => {
  const browser = await chromium.launch();
  const page = await browser.newPage();
  
  const routes = ['/', '/crm', '/inbox', '/quotes', '/tasks', '/analytics'];
  
  for (const route of routes) {
    console.log("Scraping " + route);
    await page.goto('https://ainexstudio.web.app' + route, { waitUntil: 'networkidle' });
    const html = await page.content();
    const name = route === '/' ? 'index' : route.replace('/', '');
    fs.writeFileSync(`ainex_${name}.html`, html);
  }
  
  await browser.close();
})();

File: scrape_ainex.py
from playwright.sync_api import sync_playwright

with sync_playwright() as p:
    browser = p.chromium.launch()
    page = browser.new_page()
    routes = ['/', '/crm', '/inbox', '/quotes', '/tasks', '/analytics']
    
    for route in routes:
        print(f"Scraping {route}")
        page.goto(f"https://intro_showstudio.web.app{route}", wait_until='networkidle')
        html = page.content()
        name = 'index' if route == '/' else route.replace('/', '')
        with open(f"intro_show_{name}.html", 'w', encoding='utf-8') as f:
            f.write(html)
            
    browser.close()
